docs/src/2.install.stories.mdx

```diff
@@ -11,7 +11,15 @@ npm install mdx-embed --save
 ### Install peer dependencies
 
 ```js
-npm install @mdx-js/mdx @mdx-js/react --save
+npm install  mdx-embed @mdx-js/mdx @mdx-js/react --save
 ```
 
-... add some notes about how to use the `MDXEmbedProvider`
+Add the `MDXEmbedProvider` to your application by importing it from `mdx-embed` and wrapping your applications `MDXRenderer`.
+
+```javascript
+import { MDXEmbedProvider } from 'mdx-embed';
+
+<MDXEmbedProvider>
+  <MDXRenderer>{body}</MDXRenderer>
```

Comment:
@molebox @spences10 this isn't right is it? `MDXRenderer ` comes from the `gatsby-plugin-mdx` doesn't it? 

How are you using it in your Next demo @molebox

Reply:
MDXRenderer comes form the gatsby-plugin-mdx. You shouldnt need that, The MDXEmbedProvider should pass all its good shit down

Reply:
I think so @PaulieScanlon